CMakeLists.txt

```diff
@@ -38,6 +66,7 @@ add_subdirectory(tests)
 # Documentation
 set(COMMON_PROJECT_DOMAIN ch.epfl.bluebrain)
 include(DoxygenRule)
+include(PackageConfig)
```

Comment:
It's needed for an integration into a superproject. Otherwise, DisplayCluster cannot be found, while doing git_subproject().